SU2_CFD/src/drivers/CDriver.cpp

```diff
@@ -4178,6 +4178,19 @@ bool CFluidDriver::Monitor(unsigned long ExtIter) {
 
 void CFluidDriver::Output(unsigned long InnerIter) {
 
+  for (iZone = 0; iZone < nZone; iZone++) {
+    const auto inst = config_container[iZone]->GetiInst();
```

Comment:
Is there a reason to use `auto` as opposed to a specific type?

Also were different instantiations just not being output before you added these lines?

Reply:
Mostly because I find boring to write `unsigned blabla` all the time. It is a recommend practice in the book Effective Modern C++ (but for other reasons).

Neither zones nor instances were outputted before.

Reply:
is this still a problem in the `SingleZoneDriver` and `MultiZoneDriver` as well? 

Its not really related to this PR, but might be worth addressing

Reply:
To my knowledge only the HBDriver deals with multiple time instances... so it should not be a problem for anything else.